about.html

```diff
@@ -0,0 +1,124 @@
+                <ul class="nav-items">
+                    <li class="nav-list"><a href="index.html">Home</a></li>
+                    <li class="nav-list"><a href="about.html">Program</a></li>
+                    <li class="nav-list"><a href="#">Join</a></li>
+                    <li class="nav-list"><a href="#">Sponsor </a></li>
+                    <li class="nav-list"><a href="#">News</a></li>
+                </ul>
+            </nav>
+
+    <section class="sec1" id="sec1">
+          <button class="btn"><i class="fa-solid fa-bars navbtn"></i></button>
+        <div class="desktopdarkheader">
+            <div class="dark-items">
+                <ul>
+                    <li><i class="fa-brands fa-facebook-f"></i></li>
+                    <li><img src="images/twitter.png" alt=""></li>
+                    <li>English</li>
+                    <li>My Page</li>
+                    <li>Logout</li>
+                </ul>
+            </div>
+        </div>
+        <div class="desktoplighterheader">
+            <div class="desktopheader">AALA 2022</div>
+
+            <nav class="references">
+                <a href="about.html" class="about" id="refitem">About</a>
+                <a href="#mainprogramme" class="programme">Program</a>
+                <a href="" class="join">Join</a>
+                <a href="#sec4" class="sponsor">Sponsor</a>
+                <a href="" class="news">News</a>
+            </nav>
+
+        </div>
+        
+        <div class="about-container">
+            <div id="about-container">
+                <div id="about-description">
+                    <h1 class="about-description">Annual Art And Literary Appreciation 2022!</h1>
+                </div>
+                <div id="about-texts"><br>The Annual Art And Literary Appreciation brings together a community of
+                    experts, academics, and musicians who comprise the AALA affiliate
+                    network across the globe. <br>This year we hope to reach a new milestone where our experts offer
+                    mentorship tofour lucky mentees</br><br>so, this year, if you hope and are open to achieving
+                    something really huge in literary Appreciation, music, arts, culture, and history this summit is for
+                    you -</br>
+                    <br>We hope you'll consider joining us this year to build experience and learn from the best whilst
+                    you foster your own growth
+                    </p>
+                </div>
+               <div id="contacts"><p class="contact">Please contact us via Email for any further questions about AALA 2022</p></div> 
+                <div id="contact">Hardejuwon@gmail.com</div>
+            </div>
+        </div>
+    </section>
+
+
+     <article class="sectioned2">
+        <div id="sectioned2">
+            <div id="AALA">
+                <h1 class="AALA1">Annual Art And Literary Appreciation 2022 logo</h1>
```

Comment:
Kindly note that there should only be one `h1` tag per page to follow the best practices, so kindly remove the second `h1` tag 🤗

Reply:
done thanks